todeploy/Dask_NDVI_Anomaly.ipynb

```diff
@@ -0,0 +1,4677 @@
+{
```

Comment:
<p>Should be marked as #jupyteronly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Are you sure, it runs with this cell as #jupyteronly, when I run the notebook it doesn't run without this cell. </p><p> </p><pre class="ql-syntax" spellcheck="false">name 'dc' is not defined
</pre><p> </p>

Reply:
As this is already initialised within the deployment setup, this can be moved to a jupyteronly cell as still required when running directly from the notebook.

Reply:
<p>desired changes already applied</p>